.github/workflows/advanced.yml

```diff
@@ -104,6 +104,9 @@ env:
   FFMPEG_REPO: FFmpeg/FFmpeg
   FFMPEG_BRANCH: release/6.1
 
+  BASH_REPO: Windows-on-ARM-Experiments/bash
+  BASH_BRANCH: initial-workarounds
```

Comment:
I'd also add those variables to `config.sh` and the checkout of Bash to `update-sources.sh` to the "packages" part.

Reply:
Added.